lib/presentation/organisms/timer_organism.dart

```diff
@@ -15,6 +16,7 @@ class TimerOrganism extends StatefulWidget {
 class _TimerOrganismState extends State<TimerOrganism>
     with AutomaticKeepAliveClientMixin<TimerOrganism> {
   HomeState state = HomeState.getReadyForStudy;
+  late SharedPreferences _prefs;
```

Comment:
Please use PreferencesController

Reply:
Interesting idea with `PreferencesController`!  While it sounds familiar (maybe from the Android world?). Could you elaborate a bit.

Reply:
It is a class we created in the project to use SharedPreferences

Search for it and take a look of some use cases to understand how it works

Reply:
Yeah, sure. I found it, but it didn't have `getBool` and `setBool` methods. I just added them, and now it works perfectly.